packages/firestore/lite/test/integration.test.ts

```diff
@@ -600,21 +616,53 @@ describe('deleteDoc()', () => {
   });
 });
 
-// TODO(firestorelite): Expand test coverage once we can write docs
 describe('FieldValue', () => {
   it('support equality checking with isEqual()', () => {
-    expectEqual(FieldValue.delete(), FieldValue.delete());
-    expectEqual(FieldValue.serverTimestamp(), FieldValue.serverTimestamp());
-    expectNotEqual(FieldValue.delete(), FieldValue.serverTimestamp());
-    // TODO(firestorelite): Add test when field value is available
```

Comment:
Our isEqual methods throw when instances of different types are compared, as this is not allowed by our typings. Hence, this tests is "Illegal" and it wouldn't work with the `expectNotEqual` helper.